modules/spock/src/test/groovy/org/testcontainers/spock/BrowserVncRecordingIT.groovy

```diff
@@ -0,0 +1,176 @@
+package org.testcontainers.spock
+
+import org.intellij.lang.annotations.Language
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+import spock.util.EmbeddedSpecRunner
+
+class BrowserVncRecordingIT extends Specification {
+
+    @Rule
+    TemporaryFolder temp
+
+    String recordingDir
+
+    def setup() {
+        recordingDir = temp.getRoot().getAbsolutePath()
+    }
+
+    def "retains all recordings for RECORD_ALL if successful"() {
```

Comment:
would be cool to parameterize these 3 tests into something like:
```groovy
| mode | fails | files |
| RECORD_ALL | false | BrowserWebdriverContainerIT-should+record |
| RECORD_FAILING | false | |
| RECORD_FAILING | true | BrowserWebdriverContainerIT-should+record |
```

Reply:
yes cool, I try